packages/client/src/Accounts.tsx

```diff
@@ -40,37 +112,84 @@ export default function Accounts(_props: RouteComponentProps) {
 
   return (
     <div>
-      {accounts.map(account => (
-        <section key={account.id}>
-          <header>
-            <h3>{account.email}</h3>
-            {account.loggedIn ? (
-              <Link to={`/accounts/${account.id}/dashboard`}>
-                View Conversations
-              </Link>
-            ) : (
-              <button
-                onClick={() =>
-                  authenticate({ variables: { id: account.id } }).catch(
-                    setError
-                  )
-                }
-              >
-                Log In
-              </button>
-            )}
-          </header>
-        </section>
-      ))}
-      <form onSubmit={onSubmit}>
-        <input
-          type="text"
-          name="email"
-          onChange={e => setEmailValue(e.target.value)}
-          value={emailValue}
-        />
-        <button type="submit">Add Account</button>
-      </form>
+      <div className={classes.appBarSpacer} />
+      <AppBar>
+        <Toolbar>
+          <Typography variant="h6">Poodle</Typography>
+        </Toolbar>
+      </AppBar>
+      <div className={classes.root}>
+        {accounts.map(account => (
+          <section key={account.id}>
+            <header>
```

Comment:
This `<header>` tag is not necessary. Usually a `<header>` is contains a heading, such as an `<h1>` tag.